registry-mgr return ERROR null

### Checked for duplicates

No - I haven't checked

### 🐛 Describe the bug

From In Sook Moon, PPI:

Harvest worked yesterday, but somehow today I got the following message today.
 

[SUMMARY] Reading configuration from /opt/pds/registry/harvest-4.0.4/./conf/ppi/vex-aspera4-els-pad.xml
[SUMMARY] Output directory: /tmp/harvest/out
[ERROR] null
 

I tried with another data bundle. 
 
Another issue with the registry 
 
./bin/registry-manager set-archive-status -auth /home/sysop/.auth/registry-auth.txt -es file:///home/sysop/config/ppi_mcp_prod.x ml -lidvid urn:nasa:pds:vex-aspera4-els::1.0 -status archived
 
[ERROR] null
 
 
Could you please explain what the problem is?
 

### 🕵️ Expected behavior

I expected the tool to work and change the archive status

### 📜 To Reproduce

1.
2.
3.
...


### 🖥 Environment Info

_No response_

### 📚 Version of Software Used

harvest v4.0.4

### 🩺 Test Data / Additional context

_No response_

### 🦄 Related requirements

🦄 #xyz

### ⚙️ Engineering Details

_No response_

### 🎉 Integration & Test

_No response_

--- a/src/main/java/gov/nasa/pds/validate/ReferenceIntegrityMain.java
+++ b/src/main/java/gov/nasa/pds/validate/ReferenceIntegrityMain.java
@@ -4,6 +4,7 @@ import java.io.IOException;
 import javax.xml.parsers.ParserConfigurationException;
 import org.apache.commons.cli.ParseException;
 import org.xml.sax.SAXException;
+import gov.nasa.pds.registry.common.util.ExceptionUtils;
 import gov.nasa.pds.validate.ri.CommandLineInterface;
 
 public class ReferenceIntegrityMain {
@@ -13,7 +14,7 @@ public class ReferenceIntegrityMain {
     try {
       return_code = cli.process(args);
     } catch (ParseException pe) {
-      System.err.println("[ERROR] " + pe.getMessage());
+      System.err.println("[ERROR] " + ExceptionUtils.getMessage(pe));
       cli.help();
     } catch (IOException | ParserConfigurationException | SAXException e) {
       // do nothing because they are already logged.

